### PR TITLE
Update 0.41.0 release note

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
Updated the release note for 0.41.0 release.

[skip ci]
Signed-off-by: Sreekala Gopakumar sreekala.gopakumar@ibm.com

### CODE REVIEW

**Comment by pshipton** [comment]
https://github.com/eclipse-openj9/openj9-docs/issues/1178 can be added before this is merged.

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - I have added https://github.com/eclipse-openj9/openj9-docs/issues/1178 and https://github.com/eclipse-openj9/openj9-docs/issues/1171.

File: doc/release-notes/0.41/0.41.md [hunk 1]
```diff
@@ -68,6 +68,30 @@ The following table covers notable changes in v0.41.0. Further information about
...
+<td valign="top">OpenJDK 21 and later</td>
+<td valign="top">New continuations can reuse the cached structure instead of allocating new memory for it. The continuation cache is implemented in two tiers, tier 1 and tier 2. You can set the maximum size for the two tiers with the <tt>-XX:ContinuationCache</tt> option. If an application uses more than 10000 virtual threads, setting a larger cache size might improve performance.</td>
+</tr>
+<tr>
+<td valign="top"><a href="https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/265">#265</a></td>
+<td valign="top">New <tt>-XX:[+|-]UseZlibNX</tt> option is added to control the loading of the <tt>zlibnx</tt> library in the <tt>LIBPATH</tt> environment variable.</td>
+<td valign="top">All versions (AIX&reg;)</td>
+<td valign="top">AIX system adds the <tt>zlibNX</tt> library location, if available, to the <tt>LIBPATH</tt> variable by default. But, having the <tt>zlibNX</tt> library directory location in the <tt>LIBPATH</tt> variable might cause some issues. You can disable adding of the <tt>zlibNX</tt> library location to that variable with the <tt>-XX:-UseZlibNX</tt> option.</td>
```
Comment (pshipton) [inline doc/release-notes/0.41/0.41.md:87]: The AIX system ...

Comment (Sreekala-Gopakumar) [reply doc/release-notes/0.41/0.41.md]: Done. Thanks!

**Review by pshipton — APPROVED** [review]

**Pull request merged**